app/controllers/MissionController.scala

```diff
@@ -66,7 +81,12 @@ class MissionController @Inject() (implicit val env: Environment[User, SessionAu
         val concatenated = completedMissionJsonObjects ++ incompleteMissionJsonObjects
         Future.successful(Ok(JsArray(concatenated)))
       case _ =>
-        val missions = MissionTable.selectMissions
+        val missions = MissionTable.selectMissions.filter(m => 
+          m.coverage match {
+            case None => !(~=(m.distance_ft.getOrElse(Double.PositiveInfinity), 1000.0, 0.10))
+            case _ => true
+          }
+        ) // Don't include the original 1000-ft mission
```

Comment:
Instead of having this comment at the end of the block of code, can you have it be right before the start of it?

And instead of saying "Don't include the original 1000-ft mission", say something more like "Removes first 1000-ft mission for each region, since they have been split into 2 500-ft missions". The reasons for this are that 1. it gives an indication of why the missions are being removed (which won't be clear a month from now or to someone new), and 2. the word "removes" more clearly indicates that the following code will do X, while "don't include" does not as clearly indicate that this is what the code does, and it almost sounds more like a TODO.

Also link to the github issue and maybe the PR related to this, in cases where this is done. Especially since this is such a weird fix, we really want people to know why this weird filtering is being done later on.